src/components/footer/Footer.jsx

```diff
@@ -1,3 +1,34 @@
+import { FaTwitterSquare, FaGithubSquare, FaLinkedin } from "react-icons/fa";
+import { useLocation } from "react-router-dom";
+
+import "./footer.css";
+
 export function Footer() {
-  return <>Footer</>;
+  const { pathname } = useLocation();
+
```

Comment:
- component is small and descriptive for which it is intended to do
- you can follow the suggestions of other reviewer
- rest is fine and clean